src/po/uk.cp1251.po

```diff
@@ -1,7 +1,7 @@
 #
 # Ukrainian translation for Vim
 #
-# Generated from uk.po, DO NOT EDIT
+# Original translations
```

Comment:
This looks also wrong.
This should not happen if you run `make`.

Reply:
Sorry